README.md

```diff
@@ -0,0 +1,35 @@
+# Hyperledger Transact
+
+Hyperledger Transact is a platform-agnostic library for executing transactions
+with smart contracts.
+
+The Hyperledger Transact project includes several repositories:
+
+- [This repository](https://github.com/bitwiseio/transact) contains the core
+  Transact Rust library.
+
+- The [transact-contrib](https://github.com/bitwiseio/transact-contrib)
+  repository contains examples and reference implementations.
+
+- The [transact-rfcs](https://github.com/bitwiseio/transact-rfcs) repository
+  contains RFCs (requests for comments) for proposed and approved changes to
+  Hyperledger Transact.
+
+## How to Participate
+
+We welcome contributors, both organizations and individuals, to help shape
+project direction, contribute ideas, provide use cases, and work on specific
+tools and examples.
+
+## More Information
+
+- [Rust Crate and Documentation](https://crates.io/crates/transact)
+- [Hyperledger Transact mailing list](https://lists.hyperledger.org/t/transact)
```

Comment:
This link should be https://lists.hyperledger.org/g/transact

Reply:
Fixed